src/v2/controllers/FormController.js

```diff
@@ -247,8 +247,15 @@ export default Controller.extend({
     return modelJSON;
   },
 
+  /**
+   * @param model current form model
+   * @param error any errors after user action
+   * @param form current form
+   * Handle errors that get displayed right after any user action. After such form errors widget doesn't
+   * reload or re-render, but updates the AppSate with latest remediation.
+   */
   showFormErrors(model, error, form) {
-    let errorObj, idxStateError;
+    let errorObj, idxStateError, showErrorBanner = true;
```

Comment:
nit:
I always encourage to use one variable declaration per line. :)
More on personal preference - easier to read.

Reply:
Done.